src/routes/logs.js

```diff
@@ -46,7 +44,7 @@ async function write(req, res) {
   let body = req.body;
 
   if (!body.message) {
-    return res.status(500).send('Missing log message');
+    return res.status(412).send('Missing log message');
```

Comment:
I'm not sure that `412` is appropriate in this case (see http://stackoverflow.com/questions/5369480/when-is-it-appropriate-to-respond-with-a-http-412-error). I think a simple `400` would do -- what do you think?